include/freerdp/channels/rdpdr.h

```diff
@@ -298,6 +298,7 @@ enum FILE_FS_DEVICE_FLAG
 	FILE_DEVICE_SECURE_OPEN = 0x00000100
 };
 
+#if !defined(_WIN32) || !defined(__MINGW32__)
```

Comment:
`_WIN32` should be defined by `MinGW` anyway, unnecessary change

Reply:
Just to check for Mingw only for Win32. Can reduce it actually as Mingw32 will probably not be defined by anything on Unix... It was more for safety.